perllib/FixMyStreet/App/Controller/Claims.pm

```diff
@@ -0,0 +1,81 @@
+package FixMyStreet::App::Controller::Claims;
+use Moose;
+use namespace::autoclean;
+
+BEGIN { extends 'FixMyStreet::App::Controller::Form' }
+
+use utf8;
+use FixMyStreet::App::Form::Claims;
+
+has feature => (
+    is => 'ro',
+    default => 'claims'
+);
+
+has form_class => (
+    is => 'ro',
+    default => 'FixMyStreet::App::Form::Claims',
+);
+
+has index_template => (
+    is => 'ro',
+    default => 'claims/index.html',
+);
+
+sub process_claim : Private {
+    my ($self, $c, $form) = @_;
+
+    my $data = $form->saved_data;
+
+    my $report_id = $data->{report_id};
+
+    my $detail = "";
+
+    for my $stage ( @{ $form->fields_for_display } ) {
+        next if $stage->{hide};
+        for my $field ( @{ $stage->{fields} } ) {
+            next if $field->{hide};
+            $detail .= "$field->{desc}: " . $form->format_for_display($field->{name}, $field->{value}) . "\n";
```

Comment:
As this is called both here and in template the same way, perhaps `fields_for_display` should return this as the value directly (or in another field to be used)?

Reply:
I didn't want to so that in case it turns out that we need the underlying value.

Reply:
Yeah, fair enough, just wondered if you returned format_for_display result as a `pretty` field so the template/here would be bit simpler then.

Reply:
You could use `$field->{pretty}` here as well :)